x-pack/filebeat/module/zeek/ntp/_meta/fields.yml

```diff
@@ -0,0 +1,63 @@
+- name: ntp
+  type: group
+  default_field: false
+  description: >
+    Fields exported by the Zeek NTP log.
+  fields:
+    - name: version
+      type: integer
+      description: >
+        The NTP version number (1, 2, 3, 4).
+    - name: mode
+      type: integer
+      description: >
+        The NTP mode being used.
+    - name: stratum
+      type: integer
+      description: >
+        The stratum (primary server, secondary server, etc.).
+    - name: poll
+      type: double
+      description: >
+        The maximum interval between successive messages.
```

Comment:
For `poll`, `precision`, `root_delay`, and `root_disp` can you please include a unit of measurement in the description.

Reply:
Looking at the Zeek source those appear be in seconds always. https://github.com/zeek/zeek/blob/3819146511b173513d862cb1b45e4f9bc73548ec/src/analyzer/protocol/ntp/ntp-analyzer.pac#L24-L43

Reply:
I was just about to do the updates.  beat me be mere minutes :).  Appreciate the assist.